shared/css/header.scss

```diff
@@ -155,6 +157,46 @@
     border-width: 0px;
   }
 
+  .create_options {
```

Comment:
If these are the same as above we could move them out into common CSS shared by both dashboard & pegasus?

Reply:
See https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/hamburger/hamburger.js and https://github.com/code-dot-org/code-dot-org/blob/staging/shared/css/hamburger.scss for examples of common CSS and JS for the hamburger.